salt/journal-cms/init.sls

```diff
@@ -347,17 +347,11 @@ aws-credentials-www-data:
 # TODO: this should fail, but it doesn't because drush fails silently with 0 return code
 
 migrate-content:
-{% if pillar.elife.env in ['dev'] %}
```

Comment:
It was failing before because there was reference to the legacy database in the migrate scripts and now there shouldn't be.